PSRabbitMq/Send-RabbitMqMessage.ps1

```diff
@@ -63,21 +102,51 @@
         [string]$Key,
 
         [Parameter(Mandatory=$true,ValueFromPipeline=$true)]
+        [Alias('Payload')]
         $InputObject,
 
         [switch]$Persistent,
 
         [Int32]$Depth = 2,
 
-        [PSCredential]$Credential,
+        [PSCredential]
+        [System.Management.Automation.Credential()]
+        $Credential,
 
         [System.Security.Authentication.SslProtocols]$Ssl,
 
-        [parameter(Mandatory = $false)]
-        [string]$vhost
+        [string]$vhost = '/',
+
+        [ValidateSet('application/clixml+xml','application/json','text/xml', 'text/plain')]
```

Comment:
Hmmm, I'm working on this but:
1. If I add SerializeAs as a new parameter, and content type as a different param set: It's a breaking change
2. If I add SerializeAs in the same parameter set than content type, do I automatically change ContentType if not specified in PSBoundParameter? That might break the content resolution on reception (ConvertFrom-RMQDelivery) when both SerializeAs and ContentType are specified and different. Note: In this scenario I also should default to SerializeAs = 'application/clixml+xml' to ensure backward compatibility when ContentType is set, and I should add a SerializeAs NONE, or $null so that a message can be sent without being serialized.

I feel that option 1 is clearer and simpler but breaking change.
Option 2 allows backward compatibility but is a bit nasty...

What do you think?